src/components/heatmap/Heatmap.js

```diff
@@ -517,10 +631,11 @@ const Heatmap = forwardRef((props, deckRef) => {
           width: matrixWidth,
           height: matrixHeight,
         }),
+
         new OrthographicView({
           id: 'axisLeft',
           controller: false,
-          x: (transpose ? COLOR_BAR_SIZE : 0),
```

Comment:
Do you remember why this was removed?

Reply:
and alos the one in `axisTop`

Reply:
As far as I can tell it is because the axes can be more than one `COLOR_BAR_SIZE` over or down (due to potential for multiple stacked color bars). I beleive the strategy is to have the axes at `x=0` or `y=0` and to offset the heatmap/color bars from those